src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -93,6 +93,20 @@ internal static ReadOnlySymbolTable NewDefault(IEnumerable<TexlFunction> coreFun
             return s;
         }
 
+        // Helper to create a ReadOnly symbol table around a set of core functions.
+        // Important that this is mutable so that it can be changed across engines. 
+        public SymbolTable GetMutableCopy()
```

Comment:
>GetMutableCopy [](http://example.com/codeflow?start=27&length=14)

This is just copying functions. 

We should either:
0. copy the other things. 
This actually gets tricky - it has to be a deep copy, since we don't want the mutable copy to be able to modify the original readonly table. 

1. ensure that things not being copied are empty.  (throw InvalidOperation)

2. rename to note it's just working on Functions.   

1 or 2 is probably best.

Reply:
Changed function name to GetMutableSupportedFunctionsCopy